package.json

```diff
@@ -42,7 +42,7 @@
     "morgan": "^1.9.1",
```

Comment:
Synopsys Black Duck found the following vulerabilities in the cmoponent {fix_pr_node['componentName']}:

* CVE-2020-7769 - CRITICAL severity vulnerability violates policy 'RAPID No Critical Vulns': *This affects the package nodemailer before 6.4.16. Use of crafted recipient email addresses may result in arbitrary command flag injection in sendmail transport for sending mails.* Recommended to upgrade to version v6.7.0. Direct dependency. Fix in package file 'package.json'